schema.sql

```diff
@@ -270,4 +270,9 @@ INNER JOIN species ON animals.species_id = species.id
 WHERE visits.vet_id IN (SELECT id FROM vets WHERE name = 'Maisy Smith')
 GROUP BY species.name
 ORDER BY num_visits DESC
-LIMIT 1;
\ No newline at end of file
+LIMIT 1;
+
+// Vet clinic database: database performance audit
+
```

Comment:
- [ ] It would be better to add an index to `animal_id, vet_id ` on the visits table and `email` to the owners' table to increase performance  when loading them

Reply:
Hi @youmari , thanks for pointing that out. However, I would like to let you know that I indeed indexed my database, just not in the conventional ways.

The command I used is ``create index animal_id_idx on animal_id``  and ``create index vet_id_idx on vet_id`` . The performance increase is still similar to the conventional way of creating indexes.
Thanks a lot for the review!